Automattic/jetpack
# Add/accept invite endpoint

Makes the accept invite endpoint work. We do not need the accept api endpoint from endpint to live on the Jetpack side but the following PR is made for convenient testing. Of the invite flow. 

It contains content from the following PRs. 
https://github.com/Automattic/jetpack/pull/7071
https://github.com/Automattic/jetpack/pull/7068
https://github.com/Automattic/jetpack/pull/7060

#### Changes proposed in this Pull Request:
* 

#### Testing instructions:
* Use the following calypso PR https://github.com/Automattic/wp-calypso/pull/13309
* The .com PR and load this branch. 
D5335-code and D5206-code to your .com Sandbox. 

Make sure that your Jetpack side is able to talk to you .com sandbox via the api by setting the following constant in the wp-config of your jetpack site. 

define('JETPACK__WPCOM_JSON_API_HOST', 'somethingsomething.wordpress.com' );

Do the invites work as expected? 
Sandbox the public-api so that the browser is talking to the new api endpoints that are only avilble on your sandbox. 

Send the invite and update the url to point to calypso.localhost:3000 and open it in an incognto mode. 

Does the user get added as expected?



#### Proposed changelog entry for your changes:
